### PR TITLE
Update blog example documentation

### PR DESCRIPTION
**Pull request by berndtj**
This is still a work in progress.

### CODE REVIEW

File: examples/blog/README.md [hunk 1]
```diff
@@ -3,119 +3,222 @@
...
+
 ```
-DISPATCH_HOST=<your-dispatch-host>
-DISPATCH_API_HOST=<your-dispatch-api-host>
+$ kubectl -n kong get service api-gateway-kongproxy
+NAME                    TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)                      AGE
+api-gateway-kongproxy   NodePort   10.101.3.80   <none>        80:32521/TCP,443:32696/TCP   49m
+
+$ export DISPATCH_HOST=$(minikube ip)
+$ export DISPATCH_API_URL=https://$DISPATCH_HOST:32696
```
Comment (seanhuxy) [inline examples/blog/README.md]: the URL here includes "https"

File: examples/blog/README.md [hunk 2]
```diff
@@ -3,119 +3,222 @@
...
 
-If your dispatch is locally deployed, in this step, you need the https port on which your dispatch api-gateway is hosted on.
+If your dispatch is locally deployed, in this step, you need the https port on which your dispatch api-gateway is hosted on. It should already be set as ``DISPATCH_API_URL``.
 
 ```
-export DISPATCH_API_PORT=31841
-export DISPATCH_API_URL=${DISPATCH_API_HOST}:${DISPATCH_API_PORT}
+$ echo $DISPATCH_API_HOST
 
 curl -X POST https://${DISPATCH_API_URL}/post/add -k -d '{
```
Comment (seanhuxy) [inline examples/blog/README.md]: the URL here doesn't include https:// 

could you help me update this,

**Comment by seanhuxy** [comment]
LGTM

File: examples/blog/README.md [hunk 3]
```diff
@@ -3,119 +3,222 @@
...
 Use dispatch cli to test if your images, secrets and functions are deployed correctly and ready to be used.
 
 ```
-dispatch exec post --input '{"op":"add", "post":{"id":"126", "title":"helloworld", "content":"this is a content"}}' --wait
-dispatch exec post --input '{"op":"get", "post":{"id":"126"}}' --wait
-dispatch exec post --input '{"op":"update", "post":{"id":"126", "title":"nihao", "content":"nihao"}}' --wait
-dispatch exec post --input '{"op":"list"}' --wait
-dispatch exec post --input '{"op":"delete", "post":{"id":"126"}}' --wait
+$ dispatch exec post --input '{"op":"add", "post":{"id":"126", "title":"helloworld", "content":"this is a content"}}' --wait
+$ dispatch exec post --input '{"op":"get", "post":{"id":"126"}}' --wait
```
Comment (seanhuxy) [inline examples/blog/README.md]: could you help me update it to
```
dispatch exec post --input '{"op":"get", "post": "126" }' --wait
```

File: examples/blog/README.md (hunk 1 above)
Comment (berndtj) [reply examples/blog/README.md]: fixed.

File: examples/blog/README.md [hunk 4]
```diff
@@ -3,148 +3,312 @@
 ### Dispatch
 You will need a dispatch cluster deployed and configured, please follow the quick start instruction at [dispatch](https://github.com/vmware/dispatch) repository.
 
-You will need the url of your dispatch installation.
+For minikube deployment, keep a note of the port of your dispatch host and dispatch api-gateway host. This is printed
+out at the end of the dispatch installation, but you can easily fetch it via kubectl:
```
Comment (markpeek) [inline examples/blog/README.md]: I believe it is also in ~/.dispatch/config.json which might be easier to pull out than the port numbers in kubectl.

export DISPATCH_API_URL=http://$DISPATCH_HOST:$(jq '."api-https-port"' ~/.dispatch/config.json)

File: examples/blog/README.md [hunk 5]
```diff
@@ -3,148 +3,312 @@
...
+```
 
-get ``port`` with
-```kubectl get svc -n minio```,
+To fetch the ``port`` associated with the minio service:
+```
+$ kubectl get svc -n minio
+NAME              TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+minio-minio-svc   NodePort   10.102.89.197   <none>        9000:31390/TCP   37s
+$ export MINIO_PORT=31390
```
Comment (markpeek) [inline examples/blog/README.md]: Also:
```kubectl get svc -n minio -o json | kubectl get svc -n minio -o json | jq '.items[0].spec.ports[0].nodePort'```

Comment (berndtj) [reply examples/blog/README.md]: ok... I actually was going to put that in there.  I'll do it

File: examples/blog/README.md (hunk 4 above)
Comment (berndtj) [reply examples/blog/README.md]: This is going in the next checkin.

**Pull request merged**